rfcs/079-new-online/README.md

```diff
@@ -0,0 +1,404 @@
+# Showcase recently digitised works on the Collections landing page
+
+**Last modified:** 2025-10-15T10:50:00Z
+
+## Context
+
+> The online collection is continually evolving, primarily through ongoing digitisation efforts, as well as through cataloguing of born-digital works. Users could benefit from knowing what is newly available to view and download online.
+
+See [Notion page](https://www.notion.so/wellcometrust/Showcase-recently-digitised-works-1e06687658a180439819e069fac2601f) for more details.
+
+The [New online component](https://github.com/wellcomecollection/wellcomecollection.org/issues/12303) has been built and uses hard-coded data as a first step. See [Collections landing page](https://wellcomecollection.org/collections).
+
+[`WorkItem`](https://github.com/wellcomecollection/wellcomecollection.org/blob/main/content/webapp/views/components/WorkCard/index.tsx) type expected by the component.
+NOTE: there is some flexibility here, as this was built as a temporary/POC solution. 
+
+```typescript
+type WorkItem = {
+  url: string;
+  title: string;
+  image: {
+    contentUrl: string;
+    width: number;
+    height: number;
+    alt?: string;
+  };
+  labels: { text: string }[];
+  partOf?: string;
+  contributor?: string;
+  date?: string;
+};
+```
+
+This RFC aims to describe the steps that may be taken to deliver functionality quickly while developing a more robust, long-term and automated solution. 
+
+The first stage is to add "Featured Work" as a content type in Prismic and content-pipeline/API.  
+Colleagues required: 
+- 1 BE developer (Experience or Platform)
+- 1 FE developer (Experience)
+- 1 editorial team staff (assuming they will create the Prismic docs)
+- 1 content provider (to curate the featured works, eg. Collection Information, Digital Production)
```

Comment:
I think this could be removed

Reply:
As well as the `1. New Prismic Custom Type ingested by the content pipeline and available through the content API` section. (as per [this conversation](https://github.com/wellcomecollection/docs/pull/136#discussion_r2454614216))